libs/hdf-converters/test/hdf_checker.spec.ts

```diff
@@ -13,6 +14,49 @@ function omitVersions(input: ExecJSON.Execution): Partial<ExecJSON.Execution> {
   return _.omit(input, ['version', 'platform.release', 'profiles[0].sha256']);
 }
 
+test('Test asff_mapper', () => {
+  const mapper = new ASFFMapper(
+    fs.readFileSync(
+      'sample_jsons/asff_mapper/sample_input_report/asff_sample.json',
+      {encoding: 'utf-8'}
+    )
+  );
+  const result: ExecJSON.Execution = mapper.toHdf();
+  if (result !== undefined) {
+    expect(omitVersions(result)).toEqual(
+      omitVersions(
+        JSON.parse(
+          fs.readFileSync('sample_jsons/asff_mapper/asff-hdf.json', {
+            encoding: 'utf-8'
+          })
+        )
+      )
+    );
+  }
+});
+test('Test prowler_mapper', () => {
+  const mapper = new ASFFMapper(
+    fs.readFileSync(
+      'sample_jsons/prowler_mapper/sample_input_report/prowler_sample.json',
+      {encoding: 'utf-8'}
+    ),
+    undefined,
+    {name: 'Prowler', title: 'Prowler Findings'}
+  );
+  const result: ExecJSON.Execution = mapper.toHdf();
+  if (result !== undefined) {
+    expect(omitVersions(result)).toEqual(
+      omitVersions(
+        JSON.parse(
+          fs.readFileSync('sample_jsons/prowler_mapper/prowler-hdf.json', {
+            encoding: 'utf-8'
+          })
+        )
+      )
+    );
+  }
+});
```

Comment:
Since prowler data is just ASFF this can probably be merged with the ASFF test, along with the sample JSON folders.